draft-ietf-quic-transport.md

```diff
@@ -698,18 +702,18 @@ sequence of frames, as described in {{frames}}.
 
 A sender can combine multiple QUIC packets (typically a Cryptographic Handshake
 packet and a Protected packet) into one UDP datagram.  This can reduce the
-number of UDP datagrams required to be emitted when application data can be sent
-during the handshake.  A packet with a short header does not include a length,
-so it has to be the last packet included in a UDP datagram.
+number of UDP datagrams sent when application data is sent during the handshake
```

Comment:
I think Kazuho was trying to avoid using "sent" twice in the same sentence.  Maybe change the second one to "included," "transferred," etc.?